index.js

```diff
@@ -111,6 +111,28 @@ Babel.prototype.optionsHash = function() {
       hash[key] = (typeof value === 'function') ? (value + '') : value;
     }
 
+    if (options.plugins) {
+      hash.plugins = [];
```

Comment:
In every case, the hash-for-dep of a plugin's npm module is what we need. What this PR adds, would be a complement to the hash-fpr-dep approach, not the other way around.